packages/react-native-renderer/src/ReactFabricHostConfig.js

```diff
@@ -343,6 +349,18 @@ export function shouldSetTextContent(type: string, props: Props): boolean {
 }
 
 export function getCurrentEventPriority(): * {
+  const currentEventPriority = fabricGetCurrentEventPriority();
+
+  if (currentEventPriority !== undefined) {
+    switch (currentEventPriority) {
```

Comment:
One thing we discussed is that instead of native proving priority, they could send provide the name of the event here, and in this switch we would select the priority based on event name (like DOM does).

The advantage of this approach is that we can update this map in JavaScript without needing a native app change, and the source of truth for "event name to event priority" lives inside React (leaving us free to change the model cheaply). The disadvantage is that we'd need to maintain the map here for all React Native platforms, but as soon as we change the model we would need to maintain an event priority to new model priority map anyway.

@sebmarkbage @acdlite what do you think about those two approaches?

Reply:
re: sending event name of event

This doesn't work well with event priority deduction that we have implemented in Fabric. 

I do have a somewhat hybrid system in mind that would combine event priority deduction with event priority map in JavaScript. I didn't want to overcomplicate the initial implementation and learn from this before making the next step.

The APIs are still marked as "unstable" to make it clear we are still experimenting with it.